gui/journal.lua

```diff
@@ -11,7 +11,22 @@ local table_of_contents = reqscript('internal/journal/table_of_contents')
 
 local RESIZE_MIN = {w=32, h=10}
 
-JOURNAL_PERSIST_KEY = 'journal'
+local JOURNAL_PERSIST_KEY = 'journal'
+
+local JOURNAL_WELCOME_COPY =  [=[
+Welcome to gui/journal, the chronicler's tool for Dwarf Fortress!
+
+Here, you can carve out notes, sketch your grand designs, or record the history of your fortress.
+The text you write here is saved together with your fort.
+
+For guidance on navigation and hotkeys, tap the ? button in the upper right corner.
+Happy digging!
+]=]
+
+local TOC_WELCOME_COPY =  [=[
+Start a line with # symbols and a space to create header.
+Those headers will appear here, and you can click on them to jump to them in the text.
```

Comment:
I think this could use an example, such as:
```
Start a line with # symbols and a space to create a header. For example:

# My section heading

or 

## My section subheading
```